Projects/views.py

```diff
@@ -354,3 +334,32 @@ def change_label(request):
         issue.save()
         return HttpResponse(res)
     else: return HttpResponseBadRequest()
+
+def new_issue(request):
+    print('adding issue')
+    if request.method == 'POST':
+        if request.user.is_staff:
+            print('in add issue method')
+            ti = request.POST.get('title_issue')
+            li = request.POST.get('link_issue')
+            tp = request.POST.get('title_project')
+            lp = request.POST.get('link_project')
+            lev = int(request.POST.get('level'))
+            pts = int(request.POST.get('points'))
+            Issues.objects.create(mentor=request.user,title_issue=ti,link_issue=li, title_project=tp,link_project=lp,level=lev,points=pts)
+            issues=Issues.objects.all().order_by('-id')
+            paginator = Paginator(issues, 15)  # Show 15 issues per page
+            page = request.GET.get('page', 1)
+
+            try:
+                issues = paginator.get_page(page)
+            except PageNotAnInteger:
+                issues = paginator.get_page(1)
+                # issues = paginator.get_page(1)
+            except EmptyPage:
+                issues = paginator.get_page(paginator.num_pages)
+                issues = Issues.objects.none()
+            return render(request,'Projects/home.html',{'issues':issues})
+        else:
```

Comment:
fix the else part, it should show the corresponding error on that form itself rather sending jsut a simple string on a blank paper :smile:

Reply:
This part was only for debugging. Only staff member would see the add issue button and only staff member will add issue so this else part never gets executed